CedarJavaFFI/src/objects.rs

```diff
@@ -38,6 +39,12 @@ impl<'a> JEntityTypeName<'a> {
         basename: JString<'a>,
         namespace: List<'a, JString<'a>>,
     ) -> Result<Self> {
+        let jstr_basename = env.get_string(&basename)?;
+        let basename_str = String::from(jstr_basename);
+        let mut full_type_name: Vec<String> = jstr_list_to_rust_vec(env, &namespace)?;
+        full_type_name.push(basename_str);
+        let fulll_ns_str: String = full_type_name.join("::");
```

Comment:
```suggestion
        let full_ns_str: String = full_type_name.join("::");
```

Reply:
We may also want to validate that none of the strings in `namespace` contain `::`.